docs/limitations.md

```diff
@@ -341,3 +343,39 @@ a command that generates a JSON-format template specification file that
 is useable by the Clear Containers runtime. The addition of a `spec`
 command to the Clear Containers runtime would just be duplication that
 would likely always be playing catchup with `runc`.
+
+## Other notes
+
+This section contains other useful information realated to limitations,
+warnings etc. to aid in understanding and diagnosis of what is a known
+problem, a new problem, or not a problem.
+
+### Host `rdmsr` warnings
+
+When running Clear Containers, you may witness some warnings in the host
+`dmesg` log reporting `unhandled rdmsr`, such as:
+
+```
+[  319.406575] kvm [2415]: vcpu0 unhandled rdmsr: 0x1c9
+[  319.483944] kvm [2415]: vcpu0 unhandled rdmsr: 0x64e
+[  319.483966] kvm [2415]: vcpu0 unhandled rdmsr: 0x34
+
+```
+
+These warnings are generated when the Virtual Machine, via `QEMU`, makes
+requests to read `MSR` registers from the VM/host that are not supported
+by `QEMU/KVM`. In the case of Clear Containers, these warnings are
+normally related to unsupported perf event counters.
+
+These are in fact benign warnings. Unfortunately by default they appear
```

Comment:
By default, these benign warnings appear as 'red warnings', which can be alarming to the end users and system administrators, in the dmesg logs.